core/app/languages.js

```diff
@@ -598,9 +598,9 @@ export const getLanguageISO639part2Label = (lang, translationLang = 'en') => {
 
 export const languages = R.pipe(
   R.keys,
-  R.map((lang) => ({ key: lang, value: getLanguageLabel(lang) }))
+  R.map((lang) => ({ value: lang, label: getLanguageLabel(lang) }))
```

Comment:
you can retain the key or id, maybe it is useful for some purpose

Reply:
uhm, no, the key is value, we never use a key different from the value... it will be more consistent with react-select naming convention

Reply:
If we are not going to use a key different from value maybe, but at this point, we are into the core/app/languages an maybe it is better to have 

key/id, value, label

Reply:
and how will you use the key/id then? the value in that case is the language code, that is unique already